controlpanel/frontend/jinja2/login-fail.html

```diff
@@ -14,6 +14,8 @@ <h2 class="govuk-heading-l">Your session has expired</h2>
             </a>
         </p>
 
+        <p>If you are seeing this page after multiple login attempts, you may be using your email to log into the control panel instead of using your GitHub account. Please try logging in using the Continue with GitHub button instead of filling out the email address field.</p>
```

Comment:
```suggestion
        <p>If you are seeing this page after multiple login attempts, you may be using your email to log into the Control Panel instead of using your GitHub account. Please try logging in using the Continue with GitHub button instead of filling out the email address field.</p>
```